packages/web-components/src/components/button/button.scss

```
@@ -144,3 +144,19 @@ $css--plex: true !default;
 :host(#{$prefix}-button-set[stacked]) {
   @extend .#{$prefix}--btn-set--stacked;
 }
+
+:host([data-context='data-table']) {
```

```suggestion
:host(#{$prefix}-button[data-context='data-table']) {
```
We should include the component name in the host selector for consistency..

---

added these.

---

added this